Analysis/config/2017/sources.cfg

```diff
@@ -35,11 +36,11 @@ trigger: tauTau HLT_DoubleTightChargedIsoPFTau35_Trk1_TightID_eta2p1_Reg_v HLT_D
 
 [ANA_DESC full : common]
 massWindowParams: mh 116.0 35.0 111.0 45.0
-signals: GluGluSignal_NonRes VBFSignal_NonRes GluGluSignal_Radion GluGluSignal_Graviton VBFSignal_Radion VBFSignal_Graviton
+signals: GluGluSignal_NonRes GluGluSignal_NonRes_nlo VBFSignal_NonRes GluGluSignal_Radion GluGluSignal_Graviton VBFSignal_Radion VBFSignal_Graviton
 backgrounds: TTTo2L2Nu TTToSemiLeptonic TTToHadronic DY_lo_SF DY_MC_M-10-50 Wjets VV VH EWK ST QCD ttV ttVV ttH VVV SM_Higgs
 data: Data_SingleElectron Data_SingleMuon Data_Tau
 cmb_samples: TT SM DY_lo_cmb other_bkg
-draw_sequence: data GluGluSignal_NonRes VBFSignal_NonRes DY_lo_cmb TT QCD SM other_bkg
+draw_sequence: data GluGluSignal_NonRes GluGluSignal_NonRes_nlo VBFSignal_NonRes DY_lo_cmb TT QCD SM other_bkg
```

Comment:
don't draw LO and NLO together. for the moment, keep LO only in draw_sequence (same for 2018)